applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan_for_broadcast.c

```diff
@@ -256,6 +256,169 @@ static struct bt_le_per_adv_sync_cb sync_callbacks = {
 	.term = pa_sync_terminated_cb,
 };
 
+static void pa_timer_handler(struct k_work *work)
+{
+	int ret;
+
+	if (req_recv_state != NULL) {
+		enum bt_bap_pa_state pa_state;
+
+		if (req_recv_state->pa_sync_state == BT_BAP_PA_STATE_INFO_REQ) {
+			pa_state = BT_BAP_PA_STATE_NO_PAST;
+		} else {
+			pa_state = BT_BAP_PA_STATE_FAILED;
+		}
+
+		ret = bt_bap_scan_delegator_set_pa_state(req_recv_state->src_id, pa_state);
+		if (ret) {
+			LOG_ERR("set PA state to %d failed, err = %d", pa_state, ret);
+		}
+	}
+}
+
+static K_WORK_DELAYABLE_DEFINE(pa_timer, pa_timer_handler);
+
+/**
+ * @brief	Subscribe to periodic advertising sync transfer (PAST).
+ *
+ * @param[in]	conn	Pointer to the connection object.
+ * @param[in]	pa_interval	Periodic advertising interval.
+ * @return	0 if success, error otherwise.
```

Comment:
Nit, check formatting
```suggestion
 * @param[in]	conn	        Pointer to the connection object.
 * @param[in]	pa_interval	Periodic advertising interval.
 * @return	0 if success, error otherwise.
```